gateway-service-knoxsso/src/main/java/org/apache/knox/gateway/service/knoxsso/WebSSOResource.java

```diff
@@ -99,6 +106,7 @@
   private String signatureAlgorithm = "RS256";
   private List<String> ssoExpectedparams = new ArrayList<>();
   private String clusterName;
+  private Gson gson = new Gson();
```

Comment:
Do we need gson here when we already have jackson? Looks like we don't include gson anywhere else in the project currently.

Reply:
Will switch to Jackson.